File: 58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 1/20/2026, 5:28:28 PM
class Solution {
    public int lengthOfLastWord(String s) {
        int length = 0;
        int i = s.length() - 1;

        while (i >= 0 && s.charAt(i) == ' ') {
            i--;
        }

        while (i >= 0 && s.charAt(i) != ' ') {
            length++;
            i--;
        }

        return length;

    }
}

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 1/20/2026, 5:28:21 PM
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n = s.length() - 1;
        while (n >= 0 && s[n] == ' ') n--;
        int len = 0;
        while (n >= 0 && s[n] != ' ') {
            len++;
            n--;
        }
        return len;
    }
};